public/js/components/Editor.css

```diff
@@ -35,14 +32,21 @@
   opacity: 0.3;
 }
 
-.search-bar ~ .editor-mount {
-  height: calc(100% - 58px);
+.search-bar ~ .editor-mount,
+.search-bar ~ .editor-mount .CodeMirror {
+  order: 3;
+  height: 87vh;
+}
+
+.search-bar ~ .source-footer {
+  order: 4;
 }
 
 .editor-mount,
 .CodeMirror {
+  order: 2;
   width: 100%;
-  height: 100%;
+  height: 92vh;
```

Comment:
These heights were found to be just enough to avoid extra scrollbars and allow for the editor to appear full height since it doesn't work quite right with `100%` and flexbox.